file-formats/ilmb/examples/bc_sflight.ilmb.json

```diff
@@ -0,0 +1,69 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Sample ILM Object (Flight Model)",
+    "originalLanguage": "en"
+  },
+  "destructionObject": {},
```

Comment:
Not related to this change. If "destructionObject" and "callbackClass" can be empty. I suggest to remove the "$required" annotation for these fields in a **new** change. I think we can still change it.